cmd/fetch-tsa-certs/fetch_tsa_certs.go

```diff
@@ -68,9 +80,11 @@ tinkey create-keyset --key-template ECDSA_P384 --out enc-keyset.cfg --master-key
 */
 
 var (
-	// likely the root CA
-	gcpCaParent = flag.String("gcp-ca-parent", "", "Resource path to GCP CA Service CA")
-	// key only used for fetching intermediate certificate from root and signing leaf certificate
+	// Optional root CA
+	gcpCaRoot = flag.String("gcp-ca-parent", "", "Resource path to GCP CA Service CA. If unset, the intermediate cert will be self-signed instead")
+	// optional intermediate cert validity in days
+	intermediateValidity = flag.Int("intermediate-validity", 20*365, "Days the intermediate certificate will be valid for. Default 20*365. Value will be truncated by CA if one is used.")
+	// The kms key to use for "parent" certificate (intermediate if CA is used, self-signed certificate otherwise)
 	intermediateKMSKey = flag.String("intermediate-kms-resource", "", "Resource path to the asymmetric signing KMS key for the intermediate CA, starting with gcpkms://, awskms://, azurekms:// or hashivault://")
```

Comment:
Can we rename this and document that it's either the signing key for an intermediate CA when gcpCaRoot is used, or the signing key for the self-signed root CA when provided by itself?

Reply:
yeah I was considering replacing "intermediate" with "parent" in the CLI -- and then adding notes that it's either self-signed or intermediate